src/contracts.rs

```diff
@@ -21,25 +21,45 @@ use crate::interop::{InteropContext, Interop};
 #[allow(non_snake_case)]
 pub(crate) struct ParamsOfDeploy {
     pub abi: serde_json::Value,
+    pub constructorHeader: Option<serde_json::Value>,
```

Comment:
Why it is constructorHeader?

Reply:
It contains header params for constructor call. Like `constructorParams` contains parameters for constructor itself